lisp/forge-repo.el

```diff
@@ -351,7 +352,7 @@ Return the repository identified by HOST, OWNER and NAME."
     ;; of https.
     ((forge-gitlab-repository forge-gitlab-http-repository) 'gitlab)
     (forge-github-repository    'github)
-    (forge-gitea-repository     'gittea)
+    (forge-gitea-repository     'gitea)
```

Comment:
This was a sneaky bug in the codebase.

Reply:
Thanks, I have just fixed that on master.